x-pack/plugins/ml/public/explorer/explorer_charts/explorer_charts_container.test.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import mockChartData from './__mocks__/mock_chart_data.json';
+
+// Mock TimeBuckets and mlFieldFormatService, they don't play well
+// with the jest based test setup yet.
+jest.mock('ui/time_buckets', () => ({
+  TimeBuckets: function () {
+    this.setBounds = jest.fn();
+    this.setInterval = jest.fn();
+    this.getScaledDateFormat = jest.fn();
+  }
+}));
+jest.mock('../../services/field_format_service', () => ({
+  mlFieldFormatService: {
+    getFieldFormat: jest.fn()
+  }
+}));
+
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import { mlChartTooltipService } from '../../components/chart_tooltip/chart_tooltip_service';
+
+import { ExplorerChartsContainer } from './explorer_charts_container';
+
+describe('ExplorerChartsContainer', () => {
+  const mlSelectSeverityServiceMock = {
+    state: {
+      get: () => ({
+        val: ''
+      })
+    }
+  };
+
+  const mockedGetBBox = { x: 0, y: -11.5, width: 12.1875, height: 14.5 };
+  const originalGetBBox = SVGElement.prototype.getBBox;
+  beforeEach(() => SVGElement.prototype.getBBox = () => {
```

Comment:
nit, could be `beforeEach(() => SVGElement.prototype.getBBox = () => mockedGetBBox);`